src/Resources/config/config.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/master/src/Core/System/SystemConfig/Schema/config.xsd">
+
+    <card>
+        <title>Monitoring</title>
+        <title lang="de-DE">Überwachung</title>
+
+        <input-field type="text">
```

Comment:
do you maybe want this in the symfony config? we have already one

Reply:
You mean in the SymfonyConfigCompilerPass ? What would be the benefit of passing it there?

Reply:
you can configure it using config/packages/frosh_tools.yaml. idk if you want that a shop owner can change this

Reply:
Hmm I would say so, keep the obstacle small for a simple monitoring like that.

This config/packages/frosh_tools.yaml would be great in the Readme :slightly_smiling_face: